diff --git a/web/modules/contrib/block_style_plugins/src/Plugin/BlockStyleBase.php b/web/modules/contrib/block_style_plugins/src/Plugin/BlockStyleBase.php
index df915479..79ca7c27 100644
--- a/web/modules/contrib/block_style_plugins/src/Plugin/BlockStyleBase.php
+++ b/web/modules/contrib/block_style_plugins/src/Plugin/BlockStyleBase.php
@@ -411,10 +411,14 @@ protected function getStylesFromVariables(array $variables) {
     // Load the block config entity.
     /** @var \Drupal\block\Entity\Block $block */
     $block = $this->entityTypeManager->getStorage('block')->load($variables['elements']['#id']);
-    $styles = $block->getThirdPartySetting('block_style_plugins', $this->pluginId);
+    $styles = [];
